pr_testing/run-pr-hlt-p2-timing.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ex
+echo "FAILED" > $WORKSPACE/testsResults/statusfile-hlt-p2-timing.log
+
+if [ "${SINGULARITY_IMAGE}" = "" ] ; then
+  osver=$(echo ${SCRAM_ARCH} | tr '_' '\n' | head -1 | sed 's|^[a-z][a-z]*||')
+  ls /cvmfs/singularity.opensciencegrid.org >/dev/null 2>&1 || true
+  IMG_PATH="/cvmfs/singularity.opensciencegrid.org/cmssw/cms:rhel${osver}"
+  if [ ! -e "${IMG_PATH}" ] ; then
+    IMG_PATH="/cvmfs/unpacked.cern.ch/registry.hub.docker.com/${DOCKER_IMG}"
+  fi
+  export SINGULARITY_IMAGE="${IMG_PATH}"
+fi
+
+[ "${WORKSPACE}" != "" ]         || export WORKSPACE=$(pwd) && cd $WORKSPACE
+mkdir -p $WORKSPACE/testsResults
+source $WORKSPACE/cms-bot/pr_testing/setup-pr-test-env.sh
+source $WORKSPACE/cms-bot/common/github_reports.sh
```

Comment:
I think `source $(dirname $0)/setup-pr-test-env.sh` is enough to set the cmssw env. Thr above block is not needed